Modules/Forum/classes/class.ilForumSettingsGUI.php

```diff
@@ -762,30 +762,19 @@ private function initNotificationSettingsForm() : bool
 
             $cb_grp = new ilCheckboxGroupInputGUI($this->lng->txt('notification_settings'), 'notification_events');
 
-            $notify_modified = new ilCheckboxInputGUI($this->lng->txt('notify_modified'), 'notify_modified');
-            $notify_modified->setValue((string) ilForumNotificationEvents::UPDATED);
+            $notify_modified = new ilCheckboxOption($this->lng->txt('notify_modified'), (string) ilForumNotificationEvents::UPDATED);
```

Comment:
Dind't affect this the storage of the form?

Reply:
In fact, I can test this, after the getLinkTarget-issue is fixed.